src/test/datascience/widgets/standard.vscode.test.ts

```diff
@@ -177,12 +177,12 @@ suite('Standard IPyWidget (Execution) (slow) (WIDGET_TEST)', function () {
         await kernel.interrupt();
         await testSliderWidget(comms);
 
-        // Clear all cells and restart and test again.
-        await kernel.interrupt();
-        const cell = vscodeNotebook.activeNotebookEditor?.document.cellAt(0)!;
-        await commands.executeCommand('notebook.clearAllCellsOutputs');
-        await waitForCondition(async () => cell.outputs.length === 0, 5_000, 'Cell did not get cleared');
+        // // Clear all cells and restart and test again.
```

Comment:
Does this part of the test still not work?

Reply:
looks like tests are failing, i will disabled these again